ooniprobe/Model/Database/Measurement.mm

```diff
@@ -11,6 +11,10 @@ + (NSDictionary *)defaultValuesForEntity {
     return @{@"start_time": [NSDate date]};
 }
 
++ (SRKResultSet*)notUploadedMeasurements {
+    return [[[Measurement query] where:@"is_failed = 0 AND is_uploaded = 0"] fetch];
```

Comment:
@lorenzoPrimi is there a place where the database schema is documented?

Reply:
Measurement.h